authors/apps/authentication/social.py

```diff
@@ -8,21 +8,23 @@
 
 def register_social_user(email, username):
     user = User.objects.filter(email=email)
+    password = User.objects.make_random_password()
```

Comment:
Nice Work @JamesMudidi could you specify the length/allowed chars of the password like
` passwrod = User.objects.make_random_password(length=14, allowed_chars="abcdefghjkmnpqrstuvwxyz01234567889") # zvk0hawf8m6394,